Description: add TryExec to check if okular is installed
Author: Philip Muskovac <yofel@kubuntu.org>
Bug: https://bugs.kde.org/show_bug.cgi?id=277833
Index: mobipocket-4.6.90+repack/okular/okularApplication_mobi.desktop
===================================================================
--- mobipocket-4.6.90+repack.orig/okular/okularApplication_mobi.desktop	2011-07-15 15:05:06.516333024 +0200
+++ mobipocket-4.6.90+repack/okular/okularApplication_mobi.desktop	2011-07-15 15:05:17.646333116 +0200
@@ -119,6 +119,7 @@
 GenericName[x-test]=xxDocument Viewerxx
 GenericName[zh_CN]=文档查看器
 GenericName[zh_TW]=文件檢視器
+TryExec=okular
 Exec=okular %U %i -caption "%c"
 Icon=okular
 Type=Application
